### ISSUE TITLE
`CONFLUENCE_PAT` usage?

### ISSUE THREAD

**Issue by slonopotamus** [PacoVK/antora-confluence#43]
README states it is possible to use Confluence Personal Access Token via PAT.

What I tried:
1. I went to https://id.atlassian.com/manage-profile/security/api-tokens and created a token (note that Atlassian calles it "API Tokens", not "Personal Access Token"
2. I set `CONFLUENCE_PAT` env variable with token value
3. I tried publishing docs, but always got 403

Looking into code, antora-confluence uses `CONFLUENCE_PAT` as-is for `Bearer`. However, if one reads [API Tokens docs](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/#Use-an-API-token), the proper way to use tokens is via `Basic` header, that contains username and token. For antora-confluence this means user needs to set `CONFLUENCE_USERNAME` to their username and `CONFLUENCE_PASSWORD` to Access Token.

So.

1. It would be great if antora-confluence readme documented how to use Access Tokens
2. I failed to understand how to use `CONFLUENCE_PAT` at all.

**Comment by PacoVK** [comment]
Thanks for the hint, i think i need to change the wording a bit here and rename `Basic Auth` to `API token` to be more in line with the Atlassian docs. 
I will also add a hint that personal access tokens are a feature of self-hosted instances only ([see](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html#UsingPersonalAccessTokens-CreatingPATsintheapplication)) 

**TL;DR;** 
if you are using Atlassian cloud you cannot use PAT. To limit scope, i suggest to create a technical user with minimal access rights to be used for Captain
